apps/api/prisma/schema.prisma

```diff
@@ -54,6 +59,60 @@ model PurchaseOrders {
   updated_at             DateTime?
 
   purchase_order_line_items PurchaseOrderLineItems[]
+  good_receipts GoodsReceipt[]
```

Comment:
slight inconsitency in naming here goodsreceipt vs goodreceipt

Reply:
Good catch. Resolved!